CMakeLists.txt

```diff
@@ -1,7 +1,21 @@
 cmake_minimum_required(VERSION 3.5)
 project(raspicam2)
 
-set(CMAKE_CXX_STANDARD 14)
+# Default to C99
+if(NOT CMAKE_C_STANDARD)
+  set(CMAKE_C_STANDARD 99)
+endif()
```

Comment:
Is this required? Since the node is in C++, I expected that we do not have to set any C standard.

Reply:
Most of the files in the src folder have a ".c" extension and are treated as c files.
Additionally, this is the customary configuration in ROS2: https://index.ros.org/doc/ros2/Tutorials/Ament-CMake-Documentation/#id6

Reply:
Thanks for pointing that out. Maybe it would make sense to use the ROS2 C API to implement the node at some point.

The section ["Compiler and linker options"](https://index.ros.org/doc/ros2/Tutorials/Ament-CMake-Documentation/#compiler-and-linker-options) uses that same pattern. I've never used it in that way.